fhir-search/src/main/java/com/ibm/fhir/search/util/SearchUtil.java

```diff
@@ -1259,5 +1277,84 @@ public static String buildSearchSelfUri(String requestUriString, FHIRSearchConte
          */
         return UriBuilder.builder().context(context).requestUri(requestUriString).toSearchSelfUri();
     }
+    
+    /**
+     * Return a limited subset of elements from the resource. This subset SHOULD consist solely of all supported elements 
+     * that are marked as "summary" in the base definition of the resource(s)
+     * @param resourceType
+     * @return
+     */
+    public static Set<String> getSummaryElementNames(Class<?> resourceType) {
```

Comment:
can we remove this method and use ModelSupport now?

Reply:
done